guides/release-process.md

```diff
@@ -67,22 +52,40 @@ of Cypress. You can see the progress of the test projects by opening the status
+
+2. Create or review the release-specific documentation and changelog in [cypress-documentation](https://github.com/cypress-io/cypress-documentation). If there is not already a release-specific PR open, create one.
+     - Use [`release-automations`](https://github.com/cypress-io/release-automations)'s `issues-in-release` tool to generate a starting point for the changelog, based off of ZenHub:
+        ```shell
+        cd packages/issues-in-release
+        yarn do:changelog --release <release label>
+        ```
+    - Ensure the changelog is up-to-date and has the correct date.
+    - Merge any release-specific documentation changes into the main release PR.
+    - You can view the doc's [branch deploy preview](https://github.com/cypress-io/cypress-documentation/blob/master/CONTRIBUTING.md#pull-requests) by clicking 'Details' on the PR's `netlify-cypress-docs/deploy-preview` GitHub status check.
+
+3. `develop` should contain all of the changes made in `master`. However, this occasionally may not be the case.
+   - Ensure that `master` does not have any additional commits that are not on `develop`.
+   - Ensure all auto-generated pull requests designed to merge master into develop have been successfully merged.
 
-2. If there is a new [`cypress-example-kitchensink`](https://github.com/cypress-io/cypress-example-kitchensink/releases) version, update the corresponding dependency in [`packages/example`](../packages/example) to that new version.
+4. If there is a new [`cypress-example-kitchensink`](https://github.com/cypress-io/cypress-example-kitchensink/releases) version, update the corresponding dependency in [`packages/example`](../packages/example) to that new version.
 
-3. Use the `move-binaries` script to move the binaries for `<commit sha>` from `beta` to the `desktop` folder for `<new target version>`. This also purges the cloudflare cache for this version.
+5. Once the `develop` branch is passing for all test projects with the new changes and the `linux-x64` binary is present at `https://cdn.cypress.io/beta/binary/X.Y.Z/linux-x64/<sha>/cypress.zip`, and the `linux-x64` cypress npm package is present at `https://cdn.cypress.io/beta/binary/X.Y.Z/linux-x64/<sha>/cypress.tgz`, publishing can proceed.
+
+6. Log into AWS SSO with `aws sso login --profile <name_of_profile>`. The release scripts assumes you are using the `production` profile. If you have setup your credentials under a different profile, be sure to set the `AWS_PROFILE` environment variable.
+
+7. Use the `move-binaries` script to move the binaries for `<commit sha>` from `beta` to the `desktop` folder for `<new target version>`. This also purges the cloudflare cache for this version.
 
     ```shell
     yarn move-binaries --sha <commit sha> --version <new target version>
     ```
 
+8. Validate you are logged `npm` with `npm whoami`. Otherwise log in with `npm login`. 
+
 4. Publish the new npm package under the `dev` tag, using your personal npm account.
```

Comment:
Numbering needs to continue here.

Reply:
I stopped since markdown would render the right numbers and this would caused the full doc to change 😬 I can update though.

Reply:
[cypress-io/cypress@`a3a1977` (#20438)](https://github.com/cypress-io/cypress/pull/20438/commits/a3a1977440ee9039dec5e45488b2405949e91cc1)